src/screens/Modal/XAppBrowser/styles.tsx

```diff
@@ -5,7 +5,10 @@ import StyleService from '@services/StyleService';
 
 /* Styles ==================================================================== */
 const styles = StyleService.create({
-    container: { flex: 1 },
+    container: { 
+        position:'fixed',
+        flex: 1 
+    },
     webViewContainer: {
         flex: 1,
```

Comment:
"position: fixed" should be appended to the webView container rather than the general container